src/proto/h2/client.rs

```diff
@@ -32,6 +32,10 @@ const DEFAULT_CONN_WINDOW: u32 = 1024 * 1024 * 5; // 5mb
 const DEFAULT_STREAM_WINDOW: u32 = 1024 * 1024 * 2; // 2mb
 const DEFAULT_MAX_FRAME_SIZE: u32 = 1024 * 16; // 16kb
 
+// Same as the default value in `h2`:
+// https://docs.rs/h2/0.3.3/h2/client/struct.Builder.html#method.max_concurrent_reset_streams
+const DEFAULT_MAX_CONCURRENT_RESET_STREAMS: usize = 10;
```

Comment:
Hard-coding it here might make it different in the future, ideally we just want to use a re-export of the original value?

Reply:
Fair point. I created a PR there: https://github.com/hyperium/h2/pull/543

Reply:
Decided in another comment to go with an option.